Core/Object Arts/Dolphin/MVP/Views/Common Controls/UI.ListView.cls

```diff
@@ -1894,7 +1894,7 @@ primaryColumn
 	^self allColumns first!
 
 queryCommand: aCommandQuery
-	"Update aCommandQuery to indicates how a command would be processed.
+	"Update aCommandQuery to indicate how a command would be processed.
 	if sent to the receiver. Answers whether the receiver recognised the command
 	as one of its own (this may be ignored by the command router). This need not
```

Comment:
The last sentence here, beginning "This need not be sent..", is confusing in this context as it is guidance from the superclass implementation in View as to how overrides should be implemented. It should not have been retained with the rest of the copied down content.

Reply:
So, remove the last sentence, or space it out from the previous comment?

Reply:
Remove. The comment is only relevant in the View definition.